test/moonwall.config.json

```diff
@@ -280,7 +284,8 @@
       "runScripts": ["compile-contracts.ts compile"],
       "multiThreads": true,
       "envVars": ["DEBUG_COLORS=1"],
-      "reporters": ["html", "basic"],
+      "reporters": ["basic", "html", "json"],
```

Comment:
@timbrinded I added this one but it makes the output unreadable when you execute manually (because the json is printed :( after the basic report and it is too big)